api/docs/docs/data-definitions.md

```diff
@@ -146,21 +146,21 @@ Fields:
 
 ### Beds With Covid Patients Ratio
 
-  Ratio of staffed hospital beds that are currently in use by COVID patients. For counties, this is calculated using health service area data for the corresponding area. For more information on HSAs see the [Health Service Area section](https://apidocs.covidactnow.org/data-definitions/#health-service-areas)
+  Ratio of staffed hospital beds that are currently in use by COVID patients. For counties, this is calculated using HSA-level data for the corresponding area.
 
 **Where to access**  
 * CSV column names: ``metrics.bedsWithCovidPatientsRatio``
 * JSON file fields: ``metrics.bedsWithCovidPatientsRatio``, ``metricsTimeseries.*.bedsWithCovidPatientsRatio``
 
-### Weekly Covid Admissions Per 100k Population
+### Weekly Covid Admissions Per100K
 
-  Number of COVID patients per 100k population admitted in the past week. For counties, this is calculated using health service area data for the corresponding area. For more information on HSAs see the [Health Service Area section](https://apidocs.covidactnow.org/data-definitions/#health-service-areas)
+  Number of COVID patients per 100k population admitted in the past week. For counties, this is calculated using HSA-level data for the corresponding area.
 
 **Where to access**  
 * CSV column names: ``metrics.weeklyCovidAdmissionsPer100k``
 * JSON file fields: ``metrics.weeklyCovidAdmissionsPer100k``, ``metricsTimeseries.*.weeklyCovidAdmissionsPer100k``
 
-County- and metro-level hospitalization and ICU actuals are calculated from facility-level data which, suppresses near-zero values due to privacy concerns. As a result, this data may undercount real values, particularly at low levels.
```

Comment:
This manually-added copy get clobbered when we run `./run.py api update-schemas`. Thoughts on whether we should manually re-add this each time we generate the docs, or should we just opt to remove it? Or try to work it into the `update-schemas` script...

Reply:
I'd definitely avoid manually adding it every time.  If there's not an easy straightforward way to get it auto-generated, then I'm fine with dropping it.